config/webpack/common.webpack.config.js

```diff
@@ -74,9 +74,13 @@ function getWebpackConfig(skyPagesConfig, argv = {}) {
     // The `ngDevMode` property is defined by Angular CLI during a production build.
     // We need to set it to 'false' to get the build to pass.
     // See: https://github.com/angular/angular/issues/31595#issuecomment-519083266
+    // Setting these values to `false` also addresses some payload size issues.
+    // See: https://angular.io/guide/ivy-compatibility#payload-size-debugging
     new webpack.DefinePlugin({
       'skyPagesConfig': JSON.stringify(skyPagesConfig),
-      'ngDevMode': false
+      'ngDevMode': false,
+      'ngI18nClosureMode': false,
+      'ngJitMode': false
```

Comment:
I'm not familiar with these values, but should we be setting them for all builds, including serve?  Just based on the names, it feels like they'd be dependent on build vs serve.

Reply:
Good idea. Done.